### ISSUE TITLE
Potentially wrong crime name for rule 00025.json

### ISSUE THREAD

**Issue by cryptax** [quark-engine/quark-rules#12]
Rule `00025.json` has crime name "Monitor the broadcast action events (BOOT_COMPLETED)", but the API actually checks:

```json

 "api": [
        {
            "class": "Landroid/content/Intent;",
            "method": "getAction",
            "descriptor": "()Ljava/lang/String;"
        },
        {
            "class": "Ljava/lang/String;",
            "method": "equals",
            "descriptor": "(Ljava/lang/Object;)Z"
        }
    ],
```

There is no way to know this is monitoring BOOT_COMPLETED with this rule, nor even broadcast actions, so I think the crime name should be adjusted.

**Comment by krnick** [comment]
Hi @cryptax 
Thank you for pointing out this problem :D
I will adjust it to "Monitor the general action to be performed" based on the [getAction](https://developer.android.com/reference/android/content/Intent#getAction()) description.

Would that be more clear?

**Comment by krnick** [comment]
closed in #13 .
